identity/client/e2e/pages/MappingRulesPage.ts

```diff
@@ -0,0 +1,156 @@
+      name: "Create new mapping rule",
+    });
+    this.closeCreateMappingRuleModal = this.createMappingRuleModal.getByRole("button", {
+      name: "Close",
+    });
+    this.createMappingRuleIdField = this.createMappingRuleModal.getByRole("textbox", {
+      name: "Mapping rule ID",
+    });
+    this.createMappingRuleNameField = this.createMappingRuleModal.getByRole("textbox", {
+      name: "Mapping rule name",
+    });
+    this.createMappingRuleClaimNameField = this.createMappingRuleModal.getByRole(
+      "textbox",
+      {
+        name: "Claim name",
+      },
+    );
+    this.createMappingRuleClaimValueField = this.createMappingRuleModal.getByRole(
+      "textbox",
+      {
+        name: "Claim value",
+      },
+    );
+    this.createMappingRuleModalCancelButton = this.createMappingRuleModal.getByRole(
+      "button",
+      {
+        name: "Cancel",
+      },
+    );
+    this.createMappingRuleModalCreateButton = this.createMappingRuleModal.getByRole(
+      "button",
+      {
+        name: /create a mapping rule/i,
+      },
+    );
+
+    // Edit mapping rule modal
+    this.editMappingRuleModal = page.getByRole("dialog", { name: /edit mapping rule/i });
+    this.closeEditMappingModal = this.editMappingRuleModal.getByRole("button", {
+      name: "Close",
+    });
+    this.editMappingRuleIdField = this.editMappingRuleModal.getByRole("textbox", {
+      name: "Mapping rule ID",
+    });
+    this.editMappingRuleNameField = this.editMappingRuleModal.getByRole("textbox", {
+      name: "Mapping rule name",
+    });
+    this.editMappingRuleClaimNameField = this.editMappingRuleModal.getByRole(
+      "textbox",
+      {
+        name: "Claim name",
+      },
+    );
+    this.editMappingRuleClaimValueField = this.editMappingRuleModal.getByRole(
+      "textbox",
+      {
+        name: "Claim value",
+      },
+    );
+    this.editMappingRuleModalCancelButton = this.editMappingRuleModal.getByRole(
+      "button",
+      {
+        name: "Cancel",
+      },
+    );
+    this.editMappingRuleModalUpdateButton = this.editMappingRuleModal.getByRole(
+      "button",
+      {
+        name: /update mapping rule/i,
+      },
+    );
+
+    // Delete a mapping rule modal
+    this.deleteMappingRuleModal = page.getByRole("dialog", {
+      name: /delete mapping rule/i,
+    });
+    this.closeDeleteMappingRuleModal = this.deleteMappingRuleModal.getByRole("button", {
+      name: "Close",
+    });
+    this.deleteMappingRuleModalCancelButton = this.deleteMappingRuleModal.getByRole(
+      "button",
+      {
+        name: "Cancel",
+      },
+    );
+    this.deleteMappingRuleModalDeleteButton = this.deleteMappingRuleModal.getByRole(
+      "button",
+      {
+        name: /delete mapping rule/i,
+      },
+    );
+    this.emptyState = page.getByText("You don't have any mapping rules yet");
+    this.usersNavItem = page.getByText("Users");
+  }
+
+  async navigateToMappings() {
```

Comment:
```suggestion
  async navigateToMappingRules() {
```
Don't know where this function is called from